app/models/piece.rb

```diff
@@ -18,6 +18,27 @@ def move!(x,y)
     end
   end
 
+  #Inital Capture Logic:
+  #Need game model to test id of game
+
+  #Set initial captured status to false
+  def default_captured_status
+    self.captured = false if self.captured = nil
+  end
```

Comment:
Not withstanding the above comment, there is actually a bug here.

`if self.captured = nil` should be `if self.captured == nil'.   It's easy to mix up the assignment and comparison operators.